docs/latest/user_docs.rst

```diff
@@ -19,5 +19,5 @@ Learn how to deploy, use, and operate Envoy Gateway.
   user/udp-routing
   user/grpc-routing
   user/authn
-  user/rate-limit
+  user/ratelimit
```

Comment:
curious, why this is needed ? we have been using `rate` & `limit` as different words throughout the APIs, creating API names with the `rateLimit` prefix, so logically `-` should be added here, imo

Reply:
i see the design doc using the `ratelimit` so i changed this for same

Reply:
done